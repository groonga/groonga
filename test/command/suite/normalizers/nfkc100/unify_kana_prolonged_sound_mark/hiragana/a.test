normalize \
  'NormalizerNFKC100("unify_kana_prolonged_sound_mark", true, \
                     "report_source_offset", true)' \
  "あーぁーいーぃーうーぅーえーぇーおーぉー" \
  WITH_CHECKS|WITH_TYPES
