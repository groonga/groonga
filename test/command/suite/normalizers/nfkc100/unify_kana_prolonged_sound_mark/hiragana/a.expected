normalize   'NormalizerNFKC100("unify_kana_prolonged_sound_mark", true,                      "report_source_offset", true)'   "あーぁーいーぃーうーぅーえーぇーおーぉー"   WITH_CHECKS|WITH_TYPES
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "ああぁあいいぃいううぅうええぇえおおぉお",
    "types": [
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana",
      "hiragana"
    ],
    "checks": [
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0
    ],
    "offsets": [
      0,
      3,
      6,
      9,
      12,
      15,
      18,
      21,
      24,
      27,
      30,
      33,
      36,
      39,
      42,
      45,
      48,
      51,
      54,
      57
    ]
  }
]
